ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,182 @@
+            }
+
+            List<int> firstlist = new List<int>();
+            int maxLoop = startNumber * n;
+            int[] numbersDivisblebyN = new int[0];
+
+            for (int i = 0; i < maxLoop; i++)
+            {
+                if (i > startNumber && i % n == 0)
+                {
+                    firstlist.Add(i);
+                }
+
+            }
+
+            firstlist.ToArray();
+            return firstlist[0];
+
+        }
+
+        public void ChangeNamesOfBusinessesWithNoRevenueTo_CLOSED(Business[] businesses)
+        {
+            for (int i = 0; i < businesses.Length; i++)             {                 if (businesses[i].TotalRevenue == 0 || businesses[i].TotalRevenue == null)                 {                     businesses[i].Name = "CLOSED";                 }             }         }
+
+        public bool IsAscendingOrder(int[] numbers)
+        {
+            if (numbers == null || numbers.Length == 0)
+            {
+                return false;
+            }
+
+            int NumbersOutofOrderCount = 0;
+            int currentNumber = 0;
+            int nextNumber = 0;
+
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                currentNumber = numbers[i];
+                nextNumber = numbers[i + 1];
+                if (currentNumber > nextNumber)
+                {
+                    NumbersOutofOrderCount += 1;
+                }
+            }
+
+            if (NumbersOutofOrderCount >= 1)
+            {
+                return false;
+            }
+            else
+            {
+                return true;
+            }
+        }
+
+        public int SumElementsThatFollowAnEven(int[] numbers)
+        {
+            if (numbers == null)
+            {
+                return 0;
+            }
+
+            int sum = 0;
+            int currentNumber = 0;
+            int nextNumber = 0;
+
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                currentNumber = numbers[i];
+                nextNumber = numbers[i + 1];
+                if (numbers[i] % 2 == 0)
+                {
+                    sum += nextNumber;
+                }
+            }
+            return sum;
+        }
+
+        public string TurnWordsIntoSentence(string[] words)
+        {
+            if (words == null || words.Length == 0)
+            {
+                return "";
+
+            }
+
+            int badwords = 0;
+            int goodwords = 0;
+            List<string> wordslist = new List<string>();
+
+            for (int i = 0; i < words.Length; i++)
+            {
+
+                if (words[i] == "" || words[i] == " " || words[i] == "  ")
+                {
+
+
+                    badwords += 1;
+                }
+                else
+                {
+                    wordslist.Add(words[i].Trim());
+                    goodwords += 1;
+                }
+
+            }
+            if (goodwords == 0)
```

Comment:
If `goodwords == 0`, then `wordslist.Count` is zero. We could remove the `goodwords` variable altogether and just check `if (wordslist.Count == 0)` here instead